File: iOS/Restock/Product.swift
//
//  Product.swift
//  Restock
//
//  Created by Niklas Riekenbrauck on 02.10.15.
//  Copyright © 2015 Niklas Riekenbrauck. All rights reserved.
//

import Foundation

class Product: NSObject {
    var upc: String!
    var customDescription: String?
    var name: String?
    var imageUrl: String?
    var amount: String?
    
    init(upc: String) {
        super.init()
        self.upc = upc
    }
    
}

File: Android/Restock/app/src/main/java/io/restock/android/Product.java
package io.restock.android;

/**
 * Created by danth on 10/2/2015.
 */
public class Product {

    private String upc;
    private String name;
    private String imageUrl;

    public Product(String upc, String name, String imageUrl) {
        this.upc = upc;
        this.name = name;
        this.imageUrl = imageUrl;
    }

    public String getUPC() {
        return upc;
    }

    public String getName() {
        return name;
    }

    public String getImageUrl() {
        return imageUrl;
    }

}
